src/index.js

```diff
@@ -1,26 +1,37 @@
-import React from "react";
+import React, { Component, PropTypes } from "react";
```

Comment:
you're still passing in all of React, just destructuring those so you don't have to refer to React.Component, React.PropTypes, I assume it wouldn't work if it was just

```js
import { Component, PropTypes } from "react";
```